src/data/studies/covid-studies.json

```diff
@@ -3,10 +3,12 @@
     "Name": "Outcomes Related to COVID-19 Treated With Hydroxychloroquine Among In-patients With Symptomatic Disease (ORCHID)",
     "Short Name": "ORCHID",
     "Description": "ORCHID is a multicenter, blinded, placebo-controlled, randomized clinical trial evaluating hydroxychloroquine for the treatment of adults hospitalized with COVID-19. Patients, treating clinicians, and study personnel will all be blinded to study group assignment.",
+    "dbGaP Accession ID": "phs002299.v1.p1",
     "Type": "Interventional (Clinical Trial)",
     "Link": "https://clinicaltrials.gov/ct2/show/NCT04332991",
     "Network": "PETAL Network",
     "Responsible Party": "Boyd Taylor Thompson, Massachusetts General Hospital",
-    "Status": "Completed"
+    "Status": "Completed",
+    "Other Information": "Access to ORCHID data requires data access approvals from the NLM Database of Genotypes and Phenotypes (dbGaP) system. ORCHID data is also available to those with appropriate access permission on BioLINCC (https://biodatacatalyst.nhlbi.nih.gov/data-protection), where they may download the data.",
```

Comment:
another trailing comma. please remove :)

Reply:
I edited directly and it says 'outdated' right now - the trailing comma appears to be removed from the 'Other Information' line
![image](https://user-images.githubusercontent.com/63300314/116892408-cda2b480-abfd-11eb-9106-20e21b920e66.png)